### ISSUE TITLE
Ability to get cell value by type

### ISSUE THREAD

**Issue by svip** [qax-os/excelize#520]
It would be useful to be able to get cell values as `float64` or `time.Time` (which it is currently possible to set them as), e.g. `GetCellFloatValue` or `GetCellTimeValue`.  An ability to check type like #417, would help here.  An error could be returned if one requested the wrong type.  Particularly dates can be hard to parse, considering the date format yielded by Excelize is a bit unusual.  And it's not implausible that a user entered such a format as text, meaning some sort of code that wasn't a date.  (Honestly, why doesn't Excelize return an ISO formatted time stamp, since it doesn't actually look like the date I see in Excel (which is usually locale based anyway)?)

**Comment by xuri** [comment]
Thanks for your issue, I have added a new API `GetCellType` to support get cell data type, please upgrade to the master branch code.
